.gitignore

```diff
@@ -0,0 +1,4 @@
+/.idea
+/.bsp
+/target
```

Comment:
I think this should be `target/`

Reply:
/target will skip the whole folder i think

Reply:
target/ will skip the content inside the target.
should I change /target to target/ ?

Reply:
The intention is to skip the entire target folder. Use whichever line ignore the entire folder.